src/user-projects-hooks.tsx

```diff
@@ -81,8 +80,8 @@ export const useMakeCodeProject = () => {
     return {
       [ProjectFilenames.MainTs]: generateMainScript(gestures, "javascript"),
       [ProjectFilenames.MainBlocks]: generateMainScript(gestures, "blocks"),
-      [ProjectFilenames.CustomTs]: generateCustomTs(gestures, model),
-      [ProjectFilenames.CustomJson]: generateCustomJson(gestures),
+      [ProjectFilenames.Autogenerated]: generateCustomTs(gestures, model),
+      [ProjectFilenames.DataJson]: generateCustomJson(gestures),
```

Comment:
Maybe rename the functions as well? 
`generateCustomTs` -> `generateAutogeneratedTs`
`generateCustomJson` -> `generateCustomJson`

Reply:
Done here -> [4b068b9](https://github.com/microbit-foundation/ml-trainer/pull/306/commits/4b068b9c92d8e1150e1172986dc94c9ddff49851)